libbeat/outputs/logstash/config.go

```diff
@@ -13,6 +13,7 @@ type logstashConfig struct {
 	LoadBalance      bool                  `config:"loadbalance"`
 	BulkMaxSize      int                   `config:"bulk_max_size"`
 	Timeout          time.Duration         `config:"timeout"`
+	Pipelining       int                   `config:"pipelining"        validate:"min=0"`
```

Comment:
It's not added on purpose, as current logstash-input-beats plugin contains a bug making logstash fail miserably once enabled. Will add pipelining option to docs once it's safe to use.